.gitignore

```diff
@@ -2,6 +2,8 @@
 build/_output
 build/_test
 deploy/test
+vendor
```

Comment:
Why no longer persist the vendor folder?

Reply:
Following discussion, agree better to align with recommended approach for modules.